core/beacon/src/types.rs

```diff
@@ -84,12 +87,18 @@ impl SignedBeaconBlock {
     }
 
     pub fn genesis(shard_block_hash: CryptoHash) -> SignedBeaconBlock {
-        SignedBeaconBlock::new(0, CryptoHash::default(), vec![], shard_block_hash)
+        SignedBeaconBlock::new(0, CryptoHash::default(), 0, vec![], shard_block_hash)
+    }
+
+    pub fn signature_weight(&self) -> u128 {
+        // TODO: sum stakes instead of counting them
```

Comment:
please add a issue tracking this todo (and leave here, e.g. TODO(#123))

Reply:
Opened #279